object.js

```diff
@@ -0,0 +1,60 @@
+let movie = {
+
+    title: "mulan",
+    year: 1999,
+    director: "tony",
+    actor: "barry",
+    rating: 8,
+    timesWatched: 15,
+
+
+    recommend: function () {
+        console.log(
+            `${this.title} is movie is amazing`);
+
+    },
+    watch() {
+
+        this.timesWatched = this.timesWatched + 1
+    },
+    rating(newValue) {
+
+        this.rating = newValue
+        console.log(this.rating)
+    },
+    addactor(newactor) {
+
+        this.actor = newactor
```

Comment:
The purpose of this function is to add a new actor to the actors list. However, what you did will replace the whole actors list with the new one. You can use either the concat method or the plus sign:
`this.actors = this.actors.concat(", ", actorName);`
`this.actors = this.actors + ", " + actorName;`